crates/router/src/connector/nuvei.rs

```diff
@@ -364,7 +371,14 @@ impl ConnectorIntegration<api::Capture, types::PaymentsCaptureData, types::Payme
         &self,
         req: &types::PaymentsCaptureRouterData,
     ) -> CustomResult<Option<String>, errors::ConnectorError> {
-        let req_obj = nuvei::NuveiPaymentFlowRequest::try_from(req)?;
+        let req_obj = nuvei::NuveiPaymentFlowRequest::try_from(nuvei::NuveiPaymentRequestData {
```

Comment:
why we are creating `nuvei::NuveiPaymentRequestData` in the `get_request_body` of `ConnectorIntegration`.

It should refer from `req` only.

Reply:
@Narayanbhat166 point is why we are doing two try from.

My point is, here we are doing transformation for `Void` flow with `PaymentsCancelRouterData`. In that case we should not transform  `NuveiPaymentFlowRequest` from `NuveiPaymentRequestData`.

It should be a transformation from `PaymentsCancelRouterData` to `NuveiPaymentFlowRequest`. This a template followed for transformation.

If transformation is common then we can have common function which can be called inside transformers.